src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -184,7 +184,7 @@ public static class Parser
                           RegexOptions.IgnoreCase | RegexOptions.Compiled),
 
                 // Multi-season pack
-                new Regex(@"^(?<title>.+?)[-_. ]+(?:S|(?:Season|Saison|Series|Stagione)[_. ])(?<season>(?<!\d+)(?:\d{1,2})(?!\d+))(?:-|[-_. ]{3})(?:S|(?:Season|Saison|Series|Stagione)[_. ])?(?<season>(?<!\d+)(?:\d{1,2})(?!\d+))",
+                new Regex(@"^(?<title>.+?)(Complete Series)?[-_. ]+(?:S|(?:Season|Saison|Series|Stagione)[_. ])(?<season>(?<!\d+)(?:\d{1,2})(?!\d+))(?:-|[-_. ]{3}|\s)(?:S|(?:Season|Saison|Series|Stagione)[_. ])?(?<season>(?<!\d+)(?:\d{1,2})(?!\d+))",
```

Comment:
1. I think it might be better to change the separator from `(?:-|[-_. ]{3}|\s)` to `(?:[-_. ]{1}|[-_. ]{3}|)` that way other space-like separators in release names are included.

Reply:
refined the regex, I left out the final `|`  from your suggestion - not sure if you put that intentionally, but I was concerned that allowing no space would perhaps mess up those anime shows that are in 4 digit abs eps already.